troubleshooting/Solving v141_xp build issue.md

```diff
@@ -0,0 +1,18 @@
+### **Guide to solving v141_xp build issue in Visual Studio 2015+**
+
+
+With only the default build tools installed you might receive an error that looks like this when trying to build:
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/v141_xp-img1)
```

Comment:
Can you write a description of the image instead of saying alt text on all of them?